golos.social/objects.hpp

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <eosiolib/eosio.hpp>
+#include <common/config.hpp> // for N(x)
```

Comment:
Actually, CDT 1.4 removes `N(some)` and adds `"some"_n`. The only reason old macro used in `common/config.hpp` is tests. Testing environment needs config, but don't understand `_n` literals for names, so we need to use this workaround. Propose to undef `N()` at the end of `config.hpp`.
In other contracts you should use `_n` syntax.

*Or we all can agree to use `N()`, because it's looks better and consumes 1 char less : )*